app/components/Root/Routes.jsx

```diff
@@ -12,19 +12,14 @@ import Contacts from '../../containers/Contacts'
 import AddContact from '../../containers/AddContact'
 import EditContact from '../../containers/EditContact'
 import Settings from '../../containers/Settings'
-import DisplayWalletAccounts from '../../containers/DisplayWalletAccounts'
```

Comment:
Ahh, I think we may still need to show the account details somehow when the user is creating a wallet, but not importing a wallet.  I don't believe that experience exists in the designs though.  I can help figure out how this should work since Alf's not been available on that front.

Reply:
Resolved. 👍